junit/src/test/java/io/cucumber/junit/api/FeatureRunnerTest.java

```diff
@@ -159,11 +160,21 @@ private FeatureRunner createFeatureRunner(CucumberFeature cucumberFeature, JUnit
 
         final RuntimeOptions runtimeOptions = new RuntimeOptions(new MultiLoader(RuntimeOptions.class.getClassLoader()), Env.INSTANCE, emptyList());
 
-        final TimeService timeServiceStub = new TimeService() {
+        final Clock clockStub = new Clock() {
             @Override
-            public Instant timeInstant() {
+            public Instant instant() {
                 return Instant.EPOCH;
             }
+
+			@Override
+			public ZoneId getZone() {
+				return null;
+			}
+
+			@Override
+			public Clock withZone(ZoneId zone) {
+				return null;
+			}
```

Comment:
The indentation looks of here.

Reply:
This is weird, it shows correctly in my eclipse

Reply:
Its correct here too if I go to edit it directly in github, but the diff shown here makes it look off.

Reply:
Tabs vs spaces?

Reply:
Yea maybe that's it. I ll update it.

Reply:
Yea i think that did it. Thanks